web/tsconfig.json

```diff
@@ -42,6 +42,7 @@
         "strictNullChecks": true,
         "allowUnreachableCode": false,
         "allowUnusedLabels": false,
+        "useDefineForClassFields": false,
```

Comment:
Storybook currently requires that TSC emit the "classic" class initialization syntax, rather than the more modern.  This isn't a problem at present, and it was first introduced with Typescript 3.7 and hasn't broken anything yet, so we're sticking with it.

In the event that this becomes a major problem, we should be prepared to freeze and remove Storybook, and we should _not_ "write to the test harness."  Right now, we are not; I want to ensure we don't get into the habit of doing so.